### PR TITLE
Lesson 1.

### PR DESCRIPTION
**Pull request by SokolovskayaE**

### CODE REVIEW

File: 01-node-tutorial/answers/03-module.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Modules
+const persons = require ('./04-names')
+const sayHi = require('./05-utils');
+const data = require('./06-alternative-flavor');
+
```
Comment (mattcarlotta) [inline 01-node-tutorial/answers/03-module.js:6]: Require changes:
This file is missing a `require('./07-mind-grenade')` statement which will run the `multiplyValues()` function!

File: 01-node-tutorial/answers/09-path-module.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const path = require ('path')
+
+console.log('Directory separator:', path.sep);
+
+const filePath = path.join('C:', 'Users', 'Elena', 'Desktop', 'node-express-course', '01-node-tutorial', 'answers', '09-path-module.js');
+console.log('File directory:', filePath)
+
+const absolutePath = path.resolve(__dirname, '09-path-module.js');
```
Comment (mattcarlotta) [inline 01-node-tutorial/answers/09-path-module.js:8]: Excellent usage of a node global variable (`__dirname`)! Great job! 🥳  🎉

**Review by mattcarlotta — APPROVED** [review]
Excellent job, Elena! You nearly aced this assignment! There's just one piece of feedback below that requires a change, otherwise, your assignment is complete. Since the change is minor, I'm going to approve your PR, but you should still add the change before merging in the PR. Once again, great work! 👍

File: 01-node-tutorial/answers/03-module.js (hunk 1 above)
Comment (SokolovskayaE) [reply 01-node-tutorial/answers/03-module.js:6]: Thank you so much! I overlooked that it had to be done via 03-modules. I've already added it to the code.

File: 01-node-tutorial/answers/09-path-module.js (hunk 2 above)
Comment (SokolovskayaE) [reply 01-node-tutorial/answers/09-path-module.js:8]: Thanks a lot!

**Pull request merged**